### ISSUE TITLE
Change in behavior for default roles on  pagerduty team membership

### ISSUE THREAD

**Issue by afarbos** [terraform-providers/terraform-provider-pagerduty#192]
## Terraform Version

$ terraform --version
Terraform v0.12.24
+ provider.pagerduty v1.5.1

### Affected Resource(s)

Please list the resources as a list, for example:
- pagerduty_team_membership

### Terraform Configuration Files
```hcl
resource "pagerduty_team" "test" {
  name        = "test"
  description = "test"
}
resource "pagerduty_team_membership" "test" {
  team_id = "${pagerduty_team.test.id}"
  user_id = "test@gmail.com"
}
```

### Expected Behavior

The doc behavior, the role should be observer by default
https://www.terraform.io/docs/providers/pagerduty/r/team_membership.html#role

### Actual Behavior

Because of the following change the role is manager by default
https://github.com/terraform-providers/terraform-provider-pagerduty/commit/2a72eb4f6997f51d506ab11305d60b5bae8c6210

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

**Comment by tecnobrat** [comment]
I did make a PR just now to update the docs, I assume its just the docs that need to be changed?

https://github.com/terraform-providers/terraform-provider-pagerduty/pull/194

**Comment by hevy-CC4477** [comment]
If this is a desired behavior I would also mention it in the release also.

**Comment by stmcallister** [comment]
Appears this was fixed with PR #194, which was merged. Going to close issue.
